scripts/packages.sh

```diff
@@ -5,6 +5,4 @@ set -eu -o pipefail
 # -u: errors if an variable is referenced before being set
 # -o pipefail: causes a pipeline to produce a failure return code if any command errors
 
-# Define packages so that dependencies come first as locally
-# they will be built in the order provided
-export readonly PACKAGES=( hide-bazel-files jasmine typescript karma labs )
+export readonly PACKAGES=( create hide-bazel-files jasmine typescript karma labs )
```

Comment:
what would be needed to get rid of this file?
the release script could use a `bazel query` to iterate the things we care about

Reply:
I think it could be done with a bazel query fairly easily now that the npm_package targets are in the root workspace.